_commands/basics/gcc.md

```diff
@@ -0,0 +1,113 @@
+---
+---
+
+gcc flags
+--
+
+`gcc` is a compiler command that has the ability to execute options or 'Flags' when compiling.  These flags can be grouped and executed via the command line:
+
+`-o` places output in a filename of your choice.  If you do not use the `-o` flag the output file will have the same name as the file being compiled without any extension.
```

Comment:
Thank You!  I was going off of memory... sorry!   This change is done and I relearned it :)